sympy/geometry/ellipse.py

```diff
@@ -850,9 +850,9 @@ def minor(self):
             return ab[0]
         a, b = ab
         o = a - b < 0
-        if o == True:
+        if o:
             return a
-        elif o == False:
+        elif not o:
             return b
```

Comment:
If a and b are symbolic expressions then it is not valid to replace `if o == True` with `if o`.

Reply:
Linters may complain but there are good reasons why the code is written this way.